# Gene expression dynamics in the life cycle of the annelid *Ophelia limacina*

Authors: 

- Mikhail Greenberg, SPbU Embryology department (student)
- Vitaly Kozin, SPbU Embryology department (supervisor)
- Ilya Borisenko, SPbU Embryology department (supervisor)

## Introduction
The Spiralia group is one of the three major bilaterian clades. One of the key features of this group is its stereotypical embryonic development with early establishment of cell fates [^1] with different Spiralia species utilizing autonomous (dependent on maternal factors) and conditional (dependent on cell-cell interactions) modes of cell specification. Annelids are one of the main model taxa for studying development and evolution of Spiralia, due to of their diverse life cycles and embryonic patterns [^1].

One of the most intriguing and poorly understood aspects of Spiralian embryogenesis is zygotic genome activation (ZGA), which has been shown to act as a trigger for the initiation of complex developmental processes in other Bilateria groups [^2]. However, It is unclear how ZGA is associated with different developmental modes in Spiralia. Transcriptome profiling of early developmental stages of the unequally cleaving annelid *Platynereis* revealed that ZGA proceeds in several waves, with the major transition to the zygotic landscape coinciding with the completion of the spiral cleavage programme after autonomous specification of different cell lineages [^3][^4]. Investigating whether these developmental processes in the equally cleaving annelid *Ophelia limacina* have similar dynamics would contribute substantially to the field of evolutionary developmental biology of the Spiralia clade.

**Aim:** Investigate temporal expression patterns of developmental genes throughout the life cycle of Ophelia limacina.

**Objectives:**  
* Transcriptome assembly for different stages of *O. limacina* life cycle
* Differential gene expression analysis (DE) for *O. limacina* stages
	* Gene set enrichment analysis (GSEA) for differentially expressed genes (DEGs)
	* Identification of developmental genes in DEGs
* Identification of co-expressed gene clusters
  * Gene set enrichment analysis (GSEA) for clustered genes

### Data
*hpf - hours post fertilisation*

*dpf - days post fertilisation*

Bulk RNA-seq reads for 5 stages of *O. limacina* life cycle: 0 hpf - **unfertilised egg**, 10 hpf - **32-cell blastula**, 24 hpf - **100+ cell gastrula**, 4 dpf - **trochophore larvae**, 3 years - **adult male**.

## Workflow
### Whole workflow overview:
![](pics/workflow.png)

## Results

## References
[^1]: Henry J. Q. (2014). Spiralian model systems. The International journal of developmental biology, 58(6-8), 389–401. https://doi.org/10.1387/ijdb.140127jh
[^2]: Lee, M. T., Bonneau, A. R., & Giraldez, A. J. (2014). Zygotic genome activation during the maternal-to-zygotic transition. Annual review of cell and developmental biology, 30, 581–613. https://doi.org/10.1146/annurev-cellbio-100913-013027
[^3]: Chou, H. C., Pruitt, M. M., Bastin, B. R., & Schneider, S. Q. (2016). A transcriptional blueprint for a spiral-cleaving embryo. BMC genomics, 17, 552. https://doi.org/10.1186/s12864-016-2860-6 ↩
[^4]: Vopalensky, P., Tosches, M. A., Achim, K., Handberg-Thorsager, M., & Arendt, D. (2019). From spiral cleavage to bilateral symmetry: the developmental cell lineage of the annelid brain. BMC biology, 17(1), 81. https://doi.org/10.1186/s12915-019-0705-x 
